src/motion_input_shaping/shaped_axis.py

```diff
@@ -37,7 +45,19 @@ def __init__(self, zaber_axis: Axis, resonant_frequency: float, damping_ratio: f
             raise TypeError("Invalid Axis class was used to initialized ShapedAxis.")
```

Comment:
Is this check necessary? `zaber_axis` is an instance of the `Axis` class so is it safe to assume that it is valid since it was successfully initialized before being passed into this?

Reply:
It is possible to initialize `Axis` and `Lockstep` with an index that higher than available so this check is necessary